collect_app/src/main/java/org/odk/collect/android/utilities/ImageFileUtils.kt

```diff
@@ -28,7 +28,7 @@ object ImageFileUtils {
             else CompressFormat.JPEG
         try {
             if (bitmap != null) {
-                FileOutputStream(path).use { out -> bitmap.compress(compressFormat, 100, out) }
+                FileOutputStream(path).use { out -> bitmap.compress(compressFormat, 80, out) }
```

Comment:
There is no one recommended value I think. It depends on what you read. From what I have noticed testing the app 90 in some cases may still produce images bigger than original ones. Also I wasn't able to notice any visible difference (in quality) testing even lower values so 80 seemed to me like a good choice.

https://sirv.com/help/articles/jpeg-quality-comparison/ for example says:

```
90% JPEG quality gives a very high-quality image while gaining a significant reduction on the original 100% file size.
80% JPEG quality gives a greater file size reduction with almost no loss in quality.
75% JPEG quality and lower begins to show obvious differences in the image, which can reduce your website user experience.
```

https://www.lenspiration.com/2020/07/what-quality-setting-should-i-use-for-jpg-photos/ says:

> So, here’s my conclusion: Under “normal” circumstances, you can get by with a JPG quality setting of 80

I don't think we are abnormal an need more than 80.

Reply:
Plus this will only affect images that are scaled down either way (otherwise we don't call this method as we just copy the original file to our storage). So if someone decides to scale down they really want smaller files.

Reply:
This all sounds good. Could you pull the `80` value out as a private constant and add a comment with the notes on why it's that value?

Reply:
Ok done.